packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -301,29 +301,15 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
       const desc = model.graphqlType.description;
       const { markers, fieldNames } = metadataMap;
       if (parseMarker(markers.versioned, desc)) {
+        const metadataFields = createMetadataFields();
```

Comment:
`MetadataFields` is broad. Can we be specific here about the fields we are adding? I am thinking of `createDataSyncDatesFields`, open for other suggestions.

Reply:
What do you think about `createVersionFields` or `createVersionedFields`, a bit reluctant to put datasync stuff in schema plugin, plus this functionality is independent of datasync so might cause confusion

Reply:
Okay, works for me. 

But we have the same re-occurence of the method name in the `DataSyncPlugin.ts` file, you can bring in the suggestions contextual.